### PR TITLE
SQL: Test of COUNT aggregation (#3325)

### PR DESCRIPTION
**Pull request by VeselovAlex**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/function/aggregation/CountTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    private static final String TEST_METRIC_NAME = "m-test-sql-aggregation-count-1";
+    
+    private int total;
+    private String minDateTime;
+    private String maxDateTime;
+    
+    @BeforeClass
+    public void prepareTestSeries() throws Exception {
+        Series series = new Series(TEST_ENTITY_NAME, TEST_METRIC_NAME);
+        series.addData(new Sample("2016-10-12T00:00:00.000Z", 1));
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/function/aggregation/CountTest.java]: simplify the samples

File: src/test/java/com/axibase/tsd/api/method/sql/function/aggregation/CountTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        total = series.getData().size();
+        minDateTime = series.getData().get(0).getD();
+        maxDateTime = series.getData().get(total - 1).getD();
+        SeriesMethod.insertSeriesCheck(series);
+    }
+
+    /**
+     * #3325
+     */
+    @Test
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/function/aggregation/CountTest.java]: refactor it to parametrized   test

File: src/test/java/com/axibase/tsd/api/method/sql/function/aggregation/CountTest.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+        String[][] expectedRows = {
+                {String.valueOf(total)}
+        };
+
+        assertSqlQueryRows(sqlQuery, expectedRows);
+    }
+
+    private String selectCount(String argument) {
+        String formatString = "SELECT COUNT(%s) %n" +
```
Comment (GregoryKutuzov) [inline src/test/java/com/axibase/tsd/api/method/sql/function/aggregation/CountTest.java]: formatString -> queryTemplate

**Comment by VeselovAlex** [comment]
[x] Sample list considered OK
[x] Moved to parametrized tests
[x] formatString -> queryTemplate

Also:
[x] SQL assertions' signatures changed to JUnit-like

**Pull request merged**